Source/WebCore/svg/SVGPathElement.cpp

```diff
@@ -112,9 +111,9 @@ void SVGPathElement::attributeChanged(const QualifiedName& name, const AtomStrin
     if (name == SVGNames::dAttr) {
         auto& cache = PathSegListCache::singleton();
         if (newValue.isEmpty())
-            Ref { m_pathSegList }->baseVal()->updateByteStreamData({ });
-        else if (auto* data = cache.get(newValue))
-            Ref { m_pathSegList }->baseVal()->updateByteStreamData(*data);
+            Ref { m_pathSegList }->baseVal()->clearByteStreamData();
```

Comment:
We could keep the `updateByteSteam({ })` to avoid creating all the `clear` methods no ?

Reply:
Because `DataRef` requires Ref<Data>, `{ }` does not work well. For now, I changed it to `clearByteStreamData` since it is simpler.